lib/stsci/tools/tests/test_bitmask.py

```diff
@@ -0,0 +1,186 @@
+"""
+A module containing unit tests for the `bitmask` modue.
+
+:Authors: Mihai Cara (contact: help@stsci.edu)
+
+:License: `<http://www.stsci.edu/resources/software_hardware/pyraf/LICENSE>`_
+
+"""
+from __future__ import (absolute_import, division, unicode_literals,
+                        print_function)
+
+import warnings
+import numpy as np
+import pytest
```

Comment:
This dependency should also be declared in `setup.py` under `tests_require` for completeness (it currently does not have this).